app/models/suite.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Model to represent the middle level of facility information
+# A suite is composed of mulitple rooms, which can be bedrooms or common rooms
```

Comment:
I'll add the number of beds here (assuming that you meant common room vs bedroom by `type`)